src/Options.ts

```diff
@@ -0,0 +1,33 @@
+import { AxiosRequestConfig } from 'axios';
+import Xor from './types/Xor';
+
+type Options = Xor<
+  {
+    /**
+     * The Mollie API key, starting with `'test_'` or `'live_'`.
+     */
+    apiKey: string;
+  },
+  {
+    /**
+     * OAuth access token, starting with `'access_''.
+     */
+    accessToken: string;
+  }
+> & {
+  /**
+   * One or an array of version strings of the software you are using, such as `'RockenbergCommerce/3.1.12'`.
+   */
+  versionStrings?: string | string[];
+  /**
+   * The headers set in the requests sent to the Mollie API. `Authorization`, `User-Agent`, `Accept-Encoding`, and
+   * `Content-Type` are set by this library directly, setting them here has no effect.
+   */
+  headers?: Record<string, string>;
```

Comment:
Why did you manually add the `headers` field here, instead of `Pick<>`'ing from `AxiosRequestConfig`?

Reply:
`AxiosRequestConfig` defines `headers` as `any`, for some reason. `Record` is more specific, and it means the cloning is more predictable (see the constructor of `NetworkClient`).

Reply:
Ok, makes sense. LGTM